lib/sportradar/api/soccer/player.rb

```diff
@@ -38,17 +42,23 @@ def initialize(data)
       end
 
       def name
-        [first_name, last_name].join(' ')
+        [@first_name, @last_name].join(' ')
       end
 
       def full_name
-        full = [full_first_name, full_last_name].join(' ')
-        full == " " ? name : full
+        full = [@full_first_name, @full_last_name].join(' ')
+        full == " " ? name : @full
       end
 
       def position_name
         positions = {"G" => "Goalie", "D" => "Defender", "M" => "Midfielder", "F" => "Forward"}
-        positions[position] if position
```

Comment:
You should keep this part here still for the match_summary calls, which include the position during the game.

so maybe do an 
``` 
if position
    positions[position]
elsif teams
  ... your code .... 
end 
```